### PR TITLE
feat(words): add the words eagleye and rtklib

### PR DESCRIPTION
**Pull request by rsasaki0109**
Signed-off-by: Ryohei Sasaki <ryohei.sasaki@map4.jp>

This is the PR for integrating GNSS/IMU localizar discussed in the discussion below.
https://github.com/orgs/autowarefoundation/discussions/3257

I would like to add the words [eagleye](https://github.com/MapIV/eagleye) and [rtklib](https://github.com/tomojitakasu/RTKLIB).

Relatad PR(These PRs are still in draft stage.)
https://github.com/autowarefoundation/autoware/pull/3261
https://github.com/autowarefoundation/autoware_launch/pull/200
https://github.com/autowarefoundation/autoware.universe/pull/2848

### CODE REVIEW

**Comment by rsasaki0109** [comment]
@kenji-miyake @yukkysaito 
I would like to add the words eagleye and rtklib in integrating GNSS/IMU localizar (eagleye), is this a problem?

File: .cspell.json [hunk 1]
```diff
@@ -605,6 +605,7 @@
     "dxferp",
     "dyaw",
     "dylib",
+    "eagleye",
```
Comment (kenji-miyake) [inline .cspell.json:608]: In this case, we prefer the formal style.

```suggestion
    "Eagleye",
```

File: .cspell.json [hunk 2]
```diff
@@ -1385,6 +1386,8 @@
     "rsplit",
     "rstar",
     "rsyslog",
+    "Rtklib",
+    "rtklib",
```
Comment (kenji-miyake) [inline .cspell.json:1390]: Same here. Also, FYI, it's case-insensitive.

```suggestion
    "RTKLIB",
```

**Review by kenji-miyake — CHANGES_REQUESTED** [review]
Thank you! Added small comments. Please confirm them. :bow:

**Comment by rsasaki0109** [comment]
Thanks for your comment. Fixed.

**Review by kenji-miyake — APPROVED** [review]

**Comment by rsasaki0109** [comment]
@kenji-miyake 
I would like to re-run the following PR spell-check-differential in relation to this PR. How can I do this? ...
https://github.com/autowarefoundation/autoware/pull/3261
https://github.com/autowarefoundation/autoware/actions/runs/4130308855/jobs/7136860275

**Pull request merged**